packages/block-library/src/gallery/editor.scss

```diff
@@ -19,7 +19,7 @@ ul.wp-block-gallery {
 		outline: none;
 	}
 
-	.is-selected {
+	> .is-selected {
```

Comment:
This works and I'm fine with it. Wanted to run by you a different approach to see what you think: `figure.is-selected` (I'd also apply this to the `is-transient` rule below for clarification). Those seem to be the only two rules that don't have a block/element class along with the modifier class.

Reply:
Sure, that makes sense. I'll push a quick update to do that.